src/dev/impl/DevToys/Views/Tools/Generators/CheckSumGenerator/CheckSumGeneratorToolPage.xaml

```diff
@@ -0,0 +1,172 @@
+                VerticalAlignment="Center"
+                Margin="24">
+                <TextBlock
+                    Text="{x:Bind ViewModel.Strings.SelectFilesInstruction1}"
+                    TextTrimming="CharacterEllipsis"
+                    TextWrapping="WrapWholeWords"
+                    HorizontalAlignment="Center"/>
+                <TextBlock
+                    Text="{x:Bind ViewModel.Strings.SelectFilesInstruction2}"
+                    HorizontalAlignment="Center"/>
+                <StackPanel
+                    Orientation="Horizontal"
+                    HorizontalAlignment="Center"
+                    Spacing="6">
+                    <Button
+                        x:Name="BrowseFileButton"
+                        Style="{StaticResource AccentButtonStyle}"
+                        VerticalAlignment="Center"
+                        MinWidth="100"
+                        Content="{x:Bind ViewModel.Strings.SelectFilesInstruction3}"
+                        Command="{x:Bind ViewModel.SelectFilesBrowseCommand}"/>
+                    <TextBlock
+                        Text="/"
+                        VerticalAlignment="Center"/>
+                    <Button
+                        Style="{StaticResource AccentButtonStyle}"
+                        VerticalAlignment="Center"
+                        MinWidth="100"
+                        Content="{x:Bind ViewModel.Strings.SelectFilesInstruction4}"
+                        Command="{x:Bind ViewModel.SelectFilesPasteCommand}"/>
+                </StackPanel>
+            </StackPanel>
+        </Grid>
+
+        <Grid Grid.Row="1" Margin="0,20,0,0" Visibility="{x:Bind ViewModel.ShouldDisplayProgress, Mode=OneWay, Converter={StaticResource BooleanToVisibilityConverter}}">
+            <muxc:ProgressRing
+                Height="64"
+                Width="64"
+                IsIndeterminate="False"
+                Value="{x:Bind ViewModel.Progress, Mode=OneWay}"/>
+            <Button
+                Style="{StaticResource CommandBarFlyoutEllipsisButtonStyle}"
+                Margin="3,-1,0,0"
+                Command="{x:Bind ViewModel.CancelCommand}"
+                ToolTipService.ToolTip="{x:Bind ViewModel.Strings.Cancel}"
+                AutomationProperties.Name="{x:Bind ViewModel.Strings.Cancel}">
+                <FontIcon Glyph="&#xF75A;"/>
+            </Button>
+        </Grid>
+
+
+        <controls:CustomTextBox
+            Header="{x:Bind ViewModel.Strings.Output}"
+            IsReadOnly="True"
+            Text="{x:Bind ViewModel.Output, Mode=OneWay}"/>
+
+        <controls:CustomTextBox
+            Header="{x:Bind ViewModel.Strings.OutputComparer}"
+            Height="50"
+            AcceptsReturn="True"
+            Text="{x:Bind ViewModel.OutputComparer, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
+
+        <controlsToolKit:Loading x:Name="LoadingControl" IsLoading="{x:Bind ViewModel.ShouldDisplayProgress}"  >
```

Comment:
it seems like this isn't used. Should we remove it?

Reply:
Whoopsie! 😅
Removed the control and the import statement.